### PR TITLE
Bump rspec version to 3.3.0 for threadsafety

### PR DESCRIPTION
**Pull request by andrewvc**
This is just a good idea :). We frequently deal in threads in LS and threadsafety in rspec is something I could use today.

### CODE REVIEW

File: logstash-devutils.gemspec [hunk 1]
```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.email = ["jls@semicomplete.com"]
   spec.homepage = "https://github.com/elasticsearch/logstash-devutils"
 
-  spec.add_runtime_dependency "rspec", "~> 3.1.0" # MIT License
```
Comment (ph) [inline logstash-devutils.gemspec]: Why only 3.3, 3.4.4 is out and 3.5 is in beta?

**Comment by ph** [comment]
From the changelog: 
https://github.com/rspec/rspec-core/blob/master/Changelog.md

> Make let and subject threadsafe. (Josh Cheek, #1858)

updating from 3.1 to a newest version shouldn't be too problematic and could help with the threadsafety issues.

@andrewvc I know that the all plugins test is a bit flaky but we might want to do a sanity run before merging that? Since this changes will affect all the tests in the plugins.

**Comment by andrewvc** [comment]
@ph good call, will try that out

**Comment by andrewvc** [comment]
@ph I can't get LS to install all plugins reliably on my box. The command just hangs.

`rake test:plugins` passed with the core plugins however. Your thoughts?

**Comment by andrewvc** [comment]
@ph rebased!

**Comment by ph** [comment]
released .21

**Comment by ph** [comment]
@andrewvc I relaxed the version a bit more and released 2 new version of the gems.

**Pull request closed without merging**